python/tank/deploy/tank_commands/install.py

```diff
@@ -165,9 +167,9 @@ def run_interactive(self, log, args):
             log.info("Comment and structure preserving mode")
             log.info("-------------------------------------")
             log.info("")
-            log.info("If you add a --preserve-yaml flag, existing comments and "
```

Comment:
This messages if there three times, it needs to be refactored. Also, it uses --use-pyyaml directly instead of referencing the constants when building the string.